xml/sip_banners.xml

```diff
@@ -327,9 +295,7 @@
     <param pos="0" name="service.product" value="SIP Server"/>
     <param pos="1" name="service.version"/>
     <param pos="2" name="kamailio.platform"/>
-    <param pos="0" name="service.cpe23" value="cpe:/a:kamailio:kamailio:{service.version}"/>
   </fingerprint>
-
```

Comment:
Line breaks between fingerprints makes them much more readable, I guess you disagree?

Reply:
The automation removes it.

Reply:
OK, we can address in a future PR to auto-newline after each fingerprint, and update the automation to match, if yall agree.

Reply:
I've not issues with adding line breaks.